generic-desktop-kde-live-20160731.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "generic-desktop"

```

Comment:
I don't think so. I missed ENV-DESKTOP-kde however. Limiting to Krypton would be counter productive as we can probably expect any plasma related needle in Krypton to hit TW about 2-6 weeks later or not?